components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -399,6 +409,7 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
             )}
             {showUpdateLimitModal && (
                 <UpdateLimitModal
+                    minValue={AttributionId.parse(attributionId || "")?.kind === "user" ? 1000 : 0}
```

Comment:
Can we make the magic number `1000` a constant, please?

Reply:
Fixed ✅